embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/logging/InternalErrorLogger.kt

```diff
@@ -1,15 +1,12 @@
 package io.embrace.android.embracesdk.logging
 
-internal class InternalErrorLogger(
+import android.util.Log
+
+internal class ReportingLoggerAction(
     private val internalErrorService: InternalErrorService,
-    private val logger: InternalEmbraceLogger.LoggerAction,
     private val logStrictMode: Boolean = false
 ) : InternalEmbraceLogger.LoggerAction {
 
-    // TODO: in future we should queue these messages up and add them to the payload when the
```

Comment:
In the future we should rip the InternalErrorService to shreds but deleting this works for now